### PR TITLE
[incubator-kie-drools#5701] org.kie.api.KieBase.getRule(String, Strin…

### PR DESCRIPTION
**Pull request by tkobayas**
**Issue**:
- https://github.com/apache/incubator-kie-drools/issues/5701

`org.drools.model.codegen.execmodel.RuleAttributesTest#testMetadataBasics` is fixed.

### CODE REVIEW

File: drools-drl/drools-drl-parser/src/main/java/org/drools/drl/parser/antlr4/DRLVisitorImpl.java [hunk 1]
```diff
@@ -126,8 +126,10 @@ private void applyChildrenDescrs(PackageDescr packageDescr, List<BaseDescr> desc
             } else if (descr instanceof AttributeDescr) {
                 packageDescr.addAttribute((AttributeDescr) descr);
             } else if (descr instanceof RuleDescr) { // QueryDescr extends RuleDescr
-                packageDescr.addRule((RuleDescr) descr);
-                packageDescr.afterRuleAdded((RuleDescr) descr);
+                RuleDescr ruleDescr = (RuleDescr) descr;
+                packageDescr.addRule(ruleDescr);
+                packageDescr.afterRuleAdded(ruleDescr);
+                ruleDescr.setNamespace(packageDescr.getNamespace());
```
Comment (tkobayas) [inline drools-drl/drools-drl-parser/src/main/java/org/drools/drl/parser/antlr4/DRLVisitorImpl.java:132]: Just setting a package name to `ruleDescr.namespace`. Without this, the exec-model generated rules will be in `defaultpkg` package.

**Comment by tkobayas** [comment]
@mariofusco @yurloc @pibizza Please review, thanks!

**Review by mariofusco — APPROVED** [review]

**Review by yurloc — APPROVED** [review]
LGTM.

**Review by pibizza — APPROVED** [review]
Looks good to me

**Comment by tkobayas** [comment]
Rebased to solve a conflict.

**Comment by yurloc** [comment]
Progress: There were 549 failed tests (-7).

Fixes #5701.

**Pull request merged**